src/types.ts

```diff
@@ -151,6 +151,7 @@ export type Proposal = {
   name?: string
   description?: string
   evaluatedBlock?: Block
+  quorum: BigNumber
```

Comment:
shouldn't this be BN not BigNumber?

Reply:
At the top we type alias BN to BigNumber

Reply:
we should probably remove at some point, but sgtm!